src/types/line.js

```diff
@@ -130,6 +134,7 @@ function drawLabel(ctx, line) {
 
 	line.labelRect = {x: pos.x, y: pos.y, width, height};
 
+	ctx.beginPath();
```

Comment:
For the current use cases I'd move this in the `roundedRect` function (should remove from beginPath() from box.js then)

Reply:
Done. Thanks.